### PR TITLE
Update collaborate-in-site.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed video as it was outdated and modified steps to navigate to External collaboration settings

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2850

File: microsoft-365/solutions/collaborate-in-site.md [hunk 1]
```diff
@@ -24,28 +24,22 @@ If you need to collaborate with guests across documents, data, and lists, you ca
...
-This video shows the configuration steps described in this document.</br>
-
-> [!VIDEO https://www.microsoft.com/videoplayer/embed/RE44Llg?autoplay=false]
-
 ## Azure Organizational relationships settings
 
 Sharing in Microsoft 365 is governed at its highest level by the [organizational relationships settings in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/external-identities/delegate-invitations). If guest sharing is disabled or restricted in Azure AD, this will override any sharing settings that you configure in Microsoft 365.
 
-Check the organizational relationships settings to ensure that sharing with guests is not blocked.
+Check the External collaboration settings to ensure that sharing with guests is not blocked.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-in-site.md]:
```suggestion
Check the external collaboration settings to ensure that sharing with guests is not blocked.
```

File: microsoft-365/solutions/collaborate-in-site.md [hunk 2]
```diff
@@ -24,28 +24,22 @@ If you need to collaborate with guests across documents, data, and lists, you ca
...
 
-![Screenshot of Azure Active Directory Organizational Relationships Settings page](../media/azure-ad-organizational-relationships-settings.png)
+![Screenshot of Azure Active Directory External collaboration Settings page](../media/azure-ad-organizational-relationships-settings.png)
+
+To set External collaboration settings
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-in-site.md]:
```suggestion
To set external collaboration settings:
```

File: microsoft-365/solutions/collaborate-in-site.md [hunk 3]
```diff
@@ -24,28 +24,22 @@ If you need to collaborate with guests across documents, data, and lists, you ca
...
-To set organizational relationship settings
 
 1. Log in to Microsoft Azure at [https://portal.azure.com](https://portal.azure.com).
 2. In the left navigation, click **Azure Active Directory**.
-3. In the **Overview** pane, click **Organizational relationships**.
-4. In the **Organizational relationships** pane, click **Settings**.
-5. Ensure that **Admins and users in the guest inviter role can invite** and **Members can invite** are both set to **Yes**.
-6. If you made changes, click **Save**.
+3. Select **External Identities** and click on **External collaboration settings**.
+4. In the **Guest invite settings** pane, Ensure that **Admins and users in the guest inviter role can invite** and **Members can invite** are both set to **Yes**.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/collaborate-in-site.md]:
```suggestion
4. In the **Guest invite settings** pane, ensure that **Admins and users in the guest inviter role can invite** and **Members can invite** are both set to **Yes**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by MikePlumleyMSFT** [comment]
@msbemba thanks for updating the procedures. We would not remove the video because of a UI change. We'll put it in the queue to update.

**Comment by msbemba** [comment]
Thank you @MikePlumleyMSFT

**Pull request merged**